.github/workflows/integration_test.yml

```diff
@@ -32,7 +32,8 @@ jobs:
 
       - name: Run docker in background
         run: |
-          docker run --detach --rm --name rippled-service -p 5005:5005 -p 6006:6006 --volume "${{ github.workspace }}/.ci-config/":"/opt/ripple/etc/" --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s --env GITHUB_ACTIONS=true --env CI=true ${{ env.RIPPLED_DOCKER_IMAGE }} /opt/ripple/bin/rippled -a --conf /opt/ripple/etc/rippled.cfg
+          docker run -dit -p 5005:5005 -p 6006:6006 -v "${{ github.workspace }}/.ci-config/":/etc/opt/ripple/ --name rippled_standalone --entrypoint bash ${{ env.RIPPLED_DOCKER_IMAGE }}
+          docker exec -d rippled_standalone rippled -a
```

Comment:
Why is all this being changed?

Reply:
I'm not able to get the previous `docker run ...` command working with the latest docker image. Semantically, I've broken the existing functionality into two separate commands

I'm discussing the issue with Micheal Legleux. I don't want to stall the CI tests until its resolution.